Swen/Classes/Swen.swift

```diff
@@ -86,18 +118,25 @@ fileprivate extension Swen {
         listeners.append(listener)
         if let sticky = sticky {
             listener.queue.addOperation {
-                listener.post(sticky)
+                handler(sticky)
```

Comment:
in this way we do force dispatch, in case listener.post, if this called from the same queue as listener.queue we will have deadlock